### ISSUE TITLE
TS type definition has missing override type of "'Date'"

### ISSUE THREAD

**Issue by eaustin** [cypress-io/cypress#4090]
### Current behavior:

```
const now = new Date(2019, 3, 2).getTime();
cy.clock(now, ['Date']);  //IDE complains that only 'setTimeout', 'clearTimeout', 'setInterval', 'clearInterval' are available
```

https://docs.cypress.io/api/commands/clock.html#Now 

- documentation demonstrates that this is a valid use case

### Desired behavior:

```
const now = new Date(2019, 3, 2).getTime();
cy.clock(now, ['Date']); 
```

Should not generate error/warning message in IDE

### Versions

Cypress 3.2.0

**Comment by jeep87c** [comment]
Do you have an ETA on when this will be released? It is currently preventing us coding clock sensible based tests with cypress in a VueJs app as VueJs heavily relies on intervals.

And unfortunalety we were unable to find a way to do npm i the specific commit including this fix. If you know how, please let us now since this is blocking our team.

Thanks

**Comment by flotwig** [comment]
@jeep87c You should be able to use [`// @ts-ignore`](https://stackoverflow.com/a/43620820/3474615) before the line you're using `cy.clock()` on to suppress the warning in the meantime. 

It will be fixed with 3.3.0, which will be out soon.

**Comment by jeep87c** [comment]
Gosh I'm feeling dumb... I would swear I tried that (and also many more things...) before posting here but hey it works!

Thanks for your quick reply!
